Refactor of Tagging Section (#7516)
DataDog/documentation: content/en/logs/log_collection/java.md

</appender>
```

> ## Connect your service across Logs and APM

Reviewer: I believe we should keep the word `traces`.

We are not saying Log Management here as we are talking about the data points, not the products. So we are connecting the traces and the logs, not Log Management and APM.

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Java logging instructions][1] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.
> 
> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Java instructions][1]

**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```

Once this is done, the `ConversionPattern` to use becomes:

```xml

> <param name="ConversionPattern" value="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.env} %X{dd.service} %X{dd.version} %X{dd.trace_id} %X{dd.span_id} - %m%n" />

Reviewer: @PierreVanRoy @palazzem let's first test this and maybe take this opportunity to restore the keyvalue format with quotes for Java logs as well.

</Loggers>
```

> ## Connect your service across Logs and APM

Reviewer: I believe we should keep the word `traces` instead of APM.

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Java logging instructions][1] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Java instructions][1].
```

> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```

Once this is done, the `PatternLayout` to use becomes:

```xml

> <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.env} %X{dd.service} %X{dd.version} %X{dd.trace_id} %X{dd.span_id} - m%n" />

Reviewer: @PierreVanRoy same here

</configuration>
```

> ## Connect your service across Logs and APM

Author: ```suggestion
## Connect your service across logs and traces
```

</configuration>
```

> ## Connect your service across Logs and APM

Reviewer: We should keep `traces` instead of APM

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Java logging instructions][1] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.
> 
> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Java instructions][1].

**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```

Once this is done, the `Pattern` to use becomes:

```xml

> <Pattern>"%d{yyyy-MM-dd HH:mm:ss} %-5p %c{1}:%L - %X{dd.env} %X{dd.service} %X{dd.version} %X{dd.trace_id} %X{dd.span_id} - %m%n"</Pattern>

Reviewer: @PierreVanRoy and here :)

Once that done, edit your `logback.xml` file as described in the below `Slf4j` section.

> ## Connect your service across Logs and APM

Author: ```suggestion
## Connect your service across logs and traces
```

Once that done, edit your `logback.xml` file as described in the below `Slf4j` section.

> ## Connect your service across Logs and APM

Reviewer: Let's keep `traces`

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Java logging instructions][1] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Java instructions][1]
```

> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```

</project>
```

> ## Connect your service across Logs and APM

Author: ```suggestion
## Connect your service across logs and traces
```

</project>
```

> ## Connect your service across Logs and APM

Reviewer: Let's keep `traces`

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Java logging instructions][1] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Java instructions][1].
```

> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.
```

    </appender>
```

> ## Connect your service across Logs and APM

Author: ```suggestion
## Connect your service across logs and traces
```

    </appender>
```

> ## Connect your service across Logs and APM

Reviewer: Let's keep `traces`

> If APM is enabled for this application, the APM tracer can help connect your logs with APM data.
> See [APM Java logging instructions][2] to learn more about adding trace IDs, span IDs, and even the
> `env`, `service`, and `version` to your logs.

Reviewer: ```suggestion
If APM is enabled for this application, connect your logs and traces by automatically adding trace IDs, span IDs,
`env`, `service`, and `version` to your logs by [following the APM Java instructions][2].
```

> **Note**: If the APM tracer injects `service` into your logs, there is no need to set a `service` field in your logs configuration.

Reviewer: ```suggestion
**Note**: If the APM tracer injects `service` into your logs, it overrides the value set in the agent configuration.configuration.
```
